### PR TITLE
Added showToast on error.

### PR DESCRIPTION
**Pull request by Andu033**
See related https://github.com/openmrs/openmrs-esm-styleguide/pull/42

### CODE REVIEW

File: src/openmrs-esm-error-handling.ts [hunk 1]
```diff
@@ -1,5 +1,9 @@
+<<<<<<< HEAD
```
Comment (internettrans) [inline src/openmrs-esm-error-handling.ts]: Please fix this merge conflict.

**Review by internettrans — CHANGES_REQUESTED** [review]

File: src/openmrs-esm-error-handling.ts (hunk 1 above)
Comment (Andu033) [reply src/openmrs-esm-error-handling.ts]: Done.I Forgot to commit last change.

File: src/openmrs-esm-error-handling.ts [hunk 2]
```diff
@@ -9,6 +9,8 @@ export function handleApiError() {
   };
 }
 
-window.addEventListener("error", info => {
-  // call "showToast()" or similar inside of openmrs-esm-styleguide
-});
+window.onerror = function(info) {
+  showToast(info);
```
Comment (internettrans) [inline src/openmrs-esm-error-handling.ts]: Now that showToast takes in an object, we should update this, too. Also, the message should be a generic message that leets the user know that an error has occurred.

```suggestion
  showToast({description: "Oops! An unexpected error occurred."})
```

Comment (fatmali) [reply src/openmrs-esm-error-handling.ts]: Looks good to me with addition of @joeldenning's suggestion

**Review by internettrans — APPROVED** [review]

**Pull request merged**